webserver/views/api/v1/dataset_eval.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import absolute_import
+from flask import Blueprint, jsonify, request
+from flask_login import current_user
+from webserver.decorators import auth_required
+import db.dataset_eval
+
+bp_dataset_eval = Blueprint('api_v1_dataset_eval', __name__)
+
+@bp_dataset_eval.route("/pending-jobs-for-user", methods=["GET"])
+@auth_required
+def pending_jobs_for_user():
+    """Return a list of pending jobs for a user to be evaluated remotely"""
+    jobs = db.dataset_eval.get_pending_jobs_for_user(current_user.id)
+    jobs_list = []
+    for job in jobs:
+        jobs_list.append(job["id"])
+    return jsonify(
+        success=True,
+        jobs=jobs_list
```

Comment:
This endpoint will be used by the client to get the list of pending jobs for a user so that it can process them one by one. Soon I will upload the first version of the client using this endpoint.